addon/components/flexberry-checkbox.js

```diff
@@ -76,4 +85,32 @@ export default FlexberryBaseComponent.extend({
       checked: self.get('value'),
     });
   }),
+
+  /**
+    Called when the element of the view has been inserted into the DOM or after the view was re-rendered.
+    For more information see [didInsertElement](http://emberjs.com/api/classes/Ember.Component.html#event_didInsertElement) event of [Ember.Component](http://emberjs.com/api/classes/Ember.Component.html).
+  */
+  didInsertElement() {
+    this._super(...arguments);
+
+    // We need to select and remember DOM-element representing ui-checkbox component,
+    // but we can't just call to this.$('.flexberry-checkbox'), because ember will throw an error
+    // "You cannot access this.$() on a component with tagName: ''.".
+    // So we have to search our element in 'childViews' collection.
+    let checkboxView = this.get('childViews').find(view => {
+      let tagName = view.get('tagName');
+      if (Ember.typeOf(tagName) === 'string') {
+        tagName = tagName.trim();
+      }
+
+      return tagName !== '' && view.$().hasClass('flexberry-checkbox');
+    });
+
+    let checkboxDomElement = !Ember.isNone(checkboxView) ? checkboxView.$() : null;
+    this.set('checkboxDomElement', checkboxDomElement);
+
+    checkboxDomElement.checkbox({
```

Comment:
"Uncaught TypeError: Cannot read property 'checkbox' of null". But how it posible? If we use `flexberry-checkbox` component that we have checkboxDomElement.